models/public/deep-speech-v0.5.0/deep-speech-v0.5.0.md

```diff
@@ -0,0 +1,58 @@
+# deep-speech-v0.5.0
+
+## Use Case and High-Level Description
+
+DeepSpeech is an open source Speech-To-Text engine, using a model trained by machine learning techniques based on [Baidu's Deep Speech research paper](https://arxiv.org/abs/1412.5567). Project DeepSpeech uses Google's [TensorFlow](https://www.tensorflow.org/) to make the implementation easier.
+
+## Example
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Speech recognition                        |
+| GFlops                          | 416.97                                    |
+| MParams                         | 47.225                                    |
+| Source framework                | TensorFlow\*                              |
+
+## Performance
+
+## Input
+
+### Original Model
+
+1. Audio, name: `input_samples`, a WAVE file.
```

Comment:
There are three input layer in `model.yml` : `input_node`, `previous_state_h/read`, `previous_state_c/read`. Please specify all of them.

(also three outputs)

Reply:
Thanks for your review. I have updated the file, please check it again, thanks.